src/scenes/Event/components/EventOption/index.js

```diff
@@ -53,10 +54,12 @@ export default class EventOption extends Component {
 
 
   handleAmountBlur = ({ target: { value } }) => {
-    let { amount, consensusThreshold, onAmountChange } = this.props; // eslint-disable-line
-    [amount, consensusThreshold] = [parseInt(amount, 10), parseInt(consensusThreshold, 10)];
-    if (amount + value > consensusThreshold) {
-      onAmountChange(consensusThreshold - amount);
+    let { phase, amount, consensusThreshold, onAmountChange } = this.props; // eslint-disable-line
+    if (phase === Phases.VOTING) {
+      [amount, consensusThreshold] = [parseInt(amount, 10), parseInt(consensusThreshold, 10)];
```

Comment:
we should be parsing these to floats. the core issue that was the problem before was this situation:

- say the threshold is 100 BOT
- someone previously voted 99.99999999
- you want to vote to the threshold, but you would have to enter 0.00000001 in the amount
- the fix should be that you can enter 1 (or anything higher) and it would auto adjust to the amount (0.00000001) and let you know with some message saying `You can only vote up to the threshold, etc`

Reply:
Yeah that's what I thought we discussed and then you liked my onBlur idea where it would automatically correct it to the right amount. Right?

Reply:
I see the problem though. Fixing. Apologies

Reply:
this is now fixed